docs/README.md

```diff
@@ -0,0 +1 @@
+View the full [Reference Documentation](https://jaeddy.github.io/workflow-execution-service-schemas/docs/) for the Workflow Execution Service API.
```

Comment:
Hey, shouldn't this be this repo and not yours? @jaeddy

Reply:
Yep! Fixed. Thanks!